relay-general/src/protocol/contexts/device.rs

```diff
@@ -1,8 +1,46 @@
+use std::collections::HashMap;
+
+use once_cell::sync::Lazy;
+
 use crate::protocol::FromUserAgentInfo;
 use crate::store::user_agent::is_known;
 use crate::types::{Annotated, Object, Value};
 use crate::user_agent::{parse_device, ClientHints};
 
+/// A hashmap that translates from the android model to the more human-friendly product-names.
+/// E.g. NE2211 -> OnePlus 10 Pro 5G
+pub static ANDROID_MODEL_NAMES: Lazy<HashMap<&'static str, &'static str>> = Lazy::new(|| {
+    let mut map = HashMap::new();
+    // Note that windows paths with backslashes '\' won't work on unix systems.
+    let android_str = include_str!("../../../../files/android_models.csv");
+
+    let mut lines = android_str.lines();
+
+    let header = lines.next().expect("CSV file should have a header");
```

Comment:
i think unwraps are fine here because if it panics it will fail the tests, and the string is static so it will be the same in tests as in production